modules/querier/querier.go

```diff
@@ -663,3 +842,16 @@ func (q *Querier) searchExternalEndpoint(ctx context.Context, externalEndpoint s
 	}
 	return &searchResp, nil
 }
+
+func protoToTraceQLStatic(proto *tempopb.TraceQLStatic) traceql.Static {
```

Comment:
This feels like it belongs in a different package, but where?  The inverse of this is in the `localblocks` package.